src/routes.tsx

```diff
@@ -264,6 +266,14 @@ export const routes: RouteObject[] = [
           },
         ],
       },
+      ...(showPrivacyPolicy
```

Comment:
This does not belong to our code base. If we know that they want to add custom pages:
1) they do everything in a fork
2) we can limit routes merge conflicts by moving custom routes to separate file. Instead of this we can have sth like
 `...custom3rdPartyRoutesSlot,`

custom3rdPartyRoutesSlot.tsx 

```
// Used by 3rd parties to add custom routes to the app
export const custom3rdPartyRoutesSlot = []
```
      
their fork

```
// Used by 3rd parties to add custom routes to the root of the app
import { withDefaultTheme } from './app/components/ThemeByNetwork'
import { PrivacyPage } from './app/pages/PrivacyPage'

export const customRoutesSlots = [
   {
     path: '/privacy',
     element: withDefaultTheme(<PrivacyPage />),
   },
]
```

either way this repo should not be aware of Pontus-X custom routes.

Reply:
> we can limit routes merge conflicts by moving custom routes to separate file. 

I have done that in #1345.